src/components/shared/OktaSignInWidget/index.tsx

```diff
@@ -15,16 +15,19 @@ export default class OktaSignInWidget extends Component<
   widget: any;
 
   componentDidMount() {
-    const { onSuccess, onError } = this.props;
     this.widget = new OktaSignIn({
-      baseUrl: process.env.REACT_APP_OKTA_ISSUER,
-      clientId: process.env.REACT_APP_OKTA_CLIENT_ID,
-      redirectUri: process.env.REACT_APP_OKTA_REDIRECT_URI,
+      baseUrl: process.env.REACT_APP_OKTA_DOMAIN,
```

Comment:
I'm not too familiar with env variable setup in react. Is it possible that similar to go, we should not be accessing `env` at this level in the stack, but instead passing that info down?

Reply:
This might be a good question for @christopherhuii 
Do you know whether passing the env variables down is better practice than pulling them when we need them?

Reply:
I don't think there are any problems with directly accessing variables here. We don't have any secrets here. The usage here is to customize these variables based on the application's environment, whether it be dev, test, or prod.

https://create-react-app.dev/docs/adding-custom-environment-variables/